test/net/sourceforge/kolmafia/DataFileConsistencyTest.java

```diff
@@ -307,6 +307,181 @@ public void testValidMonstersForCombats() {
     }
   }
 
+  @Nested
+  class ModifiersDotTxt {
+    @Test
+    public void modifiersShouldApplyToValidItems() {
```

Comment:
this already exists as a nested class Modifiers way down at the bottom of the file

Reply:
Merge thing, thank you